### PR TITLE
Update docs for repairing ETL windows

### PR DESCRIPTION
**Pull request by nikovacevic**
Updates the repair endpoint URL to match changes from cascading and multiplexing.

### CODE REVIEW

File: diagnostics.md [hunk 1]
```diff
@@ -69,14 +69,14 @@ In the event of a problem, the diagnostic view would help you identify specific
...
 ```
-http://<kubecost-url>:<port>/model/etl/[allocation|assets]/rebuild?window=<RFC3339-start>,<RFC3339-end>&commit=true
+http://<kubecost-url>:<port>/model/etl/[allocation|asset]/repair?window=<RFC3339-start>,<RFC3339-end>
 ```
 
-In order to rebuild the file for the problematic date above (Note it's for _Allocation_), I would navigate to the following in a browser:
+In order to repair the file for the problematic date above (Note it's for _Allocation_), navigate to the following in a browser:
 ```
-http://<kubecost-url>:<port>/model/etl/allocation/rebuild?window=2020-11-20T00:00:00Z,2020-11-21T00:00:00Z&commit=true
+http://<kubecost-url>:<port>/model/etl/allocation/repair?window=2020-11-20T00:00:00Z,2020-11-21T00:00:00Z
```
Comment (nikovacevic) [inline diagnostics.md:79]: These are the only real changes. Everything else is linting.

Comment (AjayTripathy) [reply diagnostics.md:79]: We need to maintain some of the old stuff for pre 1.82.0 and note it for people running versions less than 1.82.0

Comment (nikovacevic) [reply diagnostics.md:79]: Do we generally do that inline, or is there some other documentation versioning idea with tagging or otherwise?

Comment (AjayTripathy) [reply diagnostics.md:79]: inline is fine

Comment (nikovacevic) [reply diagnostics.md:79]: Added a note below

**Review by AjayTripathy — APPROVED** [review]

**Pull request merged**